test-suite/src/blend.rs

```diff
@@ -159,6 +160,41 @@ test_case!(blend, async move {
             EvaluateError::ValueNotFound("noname".to_owned()).into(),
             "SELECT noname FROM BlendUser",
         ),
+        (
+            EvaluateError::ValueNotFound("noname".to_owned()).into(),
+            "SELECT * FROM BlendUser WHERE noname = 1;",
+        ),
+        (
+            EvaluateError::NestedSelectRowNotFound.into(),
+            "SELECT * FROM BlendUser WHERE id = (SELECT id FROM BlendUser WHERE id = 9);",
+        ),
+        (
+            FetchError::TableNotFound("Nothing".to_owned()).into(),
+            "SELECT * FROM Nothing;",
+        ),
+        (
+            TranslateError::TooManyTables.into(),
+            "SELECT * FROM BlendUser, BlendItem",
+        ),
+        (
+            TranslateError::UnsupportedStatement("TRUNCATE TABLE BlendUser".to_owned()).into(),
+            "TRUNCATE TABLE BlendUser;",
+        ),
+        (
+            TranslateError::UnsupportedBinaryOperator("^".to_owned()).into(),
+            "SELECT 1 ^ 2 FROM BlendUser;",
+        ),
+        (
+            TranslateError::UnsupportedQuerySetExpr(
+                "SELECT * FROM BlendItem UNION SELECT * FROM BlendItem".to_owned(),
+            )
+            .into(),
+            "SELECT * FROM BlendItem UNION SELECT * FROM BlendItem;",
+        ),
```

Comment:
These would be also.. good to find some other places.
but if it is hard for now, then.. how about using `migrate.rs`?

Reply:
good idea. However, there are still test cases with ambiguous clauses. This is the case with `migrate.rs` and `synthesize.rs`. It would be nice to have good classification criteria in the future.